tests/python/pants_test/engine/test_fs.py

```diff
@@ -357,6 +357,23 @@ def test_merge_directories(self):
 
       self.assertEquals(both_snapshot.directory_digest, both_merged)
 
+  def test_materialize_directories(self):
+    with temporary_dir() as temp_dir:
+      dir_path = os.path.join(temp_dir, "roland")
+      digest = DirectoryDigest(
+        str("63949aa823baf765eff07b946050d76ec0033144c785a94d3ebd82baa931cd16"),
+        80
+      )
+      scheduler = self.mk_scheduler(rules=create_fs_rules())
+      scheduler.materialize_directories((DirectoryToMaterialize(str(dir_path), digest),))
+      self.assertTrue(os.path.exists(dir_path))
+
+      created_file = os.path.join(dir_path, "roland")
+      self.assertTrue(os.path.isfile(created_file))
+      with open(created_file) as f:
+        content = f.read()
+        self.assertEquals(content, "European Burmese")
```

Comment:
I've got one minor assertion style thought here.

I think it'd be clearer if you left out the `exists` and `isfile` checks, since `open` will raise an exception for either of those cases.